docs/about/all/support/guidelines-for-gdpr-compliance.md

```diff
@@ -11,32 +11,28 @@ redirect_from:
 
 {% info_block errorBox %}
 
-This document conforms to the [DSGVO](https://de.wikipedia.org/wiki/Datenschutz-Grundverordnung) and [RGPD](https://fr.wikipedia.org/wiki/R%C3%A8glement_g%C3%A9n%C3%A9ral_sur_la_protection_des_donn%C3%A9es) rules.
+This document conforms to the GDPR.
 
 {% endinfo_block %}
 
-As of May the 25th, 2018, the new General Data Protection Regulations (GDPR) take effect. This document describes how the Spryker Commerce OS is compliant with the GDPR.
+This document describes how the Spryker Services supports GDPR compliance of customers’ applications.
```

Comment:
Does `Spryker Services` refer to something specific?

Reply:
Asked Liz